charger/alphatec.go

```diff
@@ -60,16 +61,21 @@ func NewAlphatecFromConfig(other map[string]interface{}) (api.Charger, error) {
 		return nil, err
 	}
 
-	return NewAlphatec(cc.URI, cc.Device, cc.Comset, cc.Baudrate, modbus.ProtocolFromRTU(cc.RTU), cc.ID, cc.Timeout)
+	return NewAlphatec(cc.URI, cc.Device, cc.Comset, cc.Baudrate, modbus.ProtocolFromRTU(cc.RTU), cc.ID, cc.Delay, cc.Timeout)
 }
 
 // NewAlphatec creates Alphatec charger
-func NewAlphatec(uri, device, comset string, baudrate int, proto modbus.Protocol, slaveID uint8, timeout time.Duration) (api.Charger, error) {
+func NewAlphatec(uri, device, comset string, baudrate int, proto modbus.Protocol, slaveID uint8, delay time.Duration, timeout time.Duration) (api.Charger, error) {
 	conn, err := modbus.NewConnection(uri, device, comset, baudrate, proto, slaveID)
 	if err != nil {
 		return nil, err
 	}
 
+	conn.Delay(100 * time.Millisecond)
```

Comment:
Default nach oben in NewAlphatecFromConfig

Reply:
Meinst du so?

Reply:
Ne 😅. Aber 2nd thought: einfach im Template reicht ja völlig. Ich finde die Doppeladministration immer sehr unglücklich.

Reply:
So. Jetzt ist es einfach fest.

Reply:
Den Timeout würde ich trotzdem gerne behalten. Der tut ja auch niemandem weh. Dann gerne 20ms Delay fix.

Reply:
Ist und war halt völlig unnötig und führt eher zu mehr Problemen durch falsche Verwendung als es bringt (nichts).